README.md

```diff
@@ -277,8 +277,9 @@ sequencer.insertSteps(index,module_name,optional_options);
 
 return value: **`sequencer`** (To allow method chaining)
 
-###Importing an independent module
-The `loadNewModule` method can be used to import a new module inside sequencer. Modules can be downloaded via npm, yarn or cdn and are imported with a custom name
+### Importing an independent module
+
+The `loadNewModule` method can be used to import a new module inside sequencer. Modules can be downloaded via npm, yarn or cdn and are imported with a custom name. If you wish to load a new module at runtime, it will need to avoid usingrequire()-- unless it is compiled with a system like browserify or webpack.
```

Comment:
small typo here -- `usingrequire` - and it can be great to use ` characters as you did with loadNewModule, right?

Reply:
Done!!